app/src/main/res/layout/card.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <androidx.cardview.widget.CardView
+        android:id="@+id/card"
+        android:layout_width="350dp"
```

Comment:
The width should probably be `match_parent` or `0 dp`, to conform to the size of the recycler view.

Reply:
Yes. Generally, any time a view is supposed to stretch from one end of the screen to the other, even if there is a margin, we should use `0dp` or `match_parent`.